css/_uswds-theme-custom-styles.scss

```diff
@@ -3044,6 +3044,12 @@ $theme-table-column-gap: 4;
   @include typeset("lang");
 }
 
+.usa-table--sticky-header thead {
+  @include at-media("desktop") {
+    top: 51px;
+  }
+}
+
```

Comment:
I noticed a weird visual quirk on desktop.

If we change this to the height of the nav `3rem` or the token equivalent it should resolve this.

I'm experiencing local quirks trying to serve this branch so this is just a quick dev tool fix. I wasn't able to test with units or the variable.

<details>

<summary>Screenshots</summary>

Current: 

![Screenshot 2023-08-15 at 9 24 00 AM](https://github.com/uswds/uswds-site/assets/25211650/0345f69b-a2c6-4a6b-b296-49522439f118)

Matching navs height:

![Screenshot 2023-08-15 at 12 47 38 PM 1](https://github.com/uswds/uswds-site/assets/25211650/e420573c-4569-47d8-9433-ccf0a7fa9ac8)

Reply:
Good catch! I updated the top value to use `$height-nav-secondary`, which is the declared height of `site-nav` in [a54e82c](https://github.com/uswds/uswds-site/pull/2229/commits/a54e82c7c3cca01d853aa3581a5915d6e098ee00)

Reply:
Great thinking! Looks great